test/e2e/e2e-test/docker/base-compose.yml

```diff
@@ -34,6 +34,8 @@ services:
       SW_STORAGE_INFLUXDB_URL: http://influxdb:8086
       JAVA_OPTS: >-
         -javaagent:/jacoco/jacocoagent.jar=classdumpdir=/jacoco/classes/oap,destfile=/jacoco/oap.exec,includes=org.apache.skywalking.*,excludes=org.apache.skywalking.oap.query.*:org.apache.skywalking.oap.server.core.query.*
+      SW_PROMETHEUS_FETCHER_ACTIVE: true
+      SW_TELEMETRY: prometheus
```

Comment:
This should be able to open self-observation. But I think we should add GraphQL query to verify the metrics really have values. NOTICE, usually, metrics value should not be 0, if there are some cases of that, we could skip those metrics, such as GC count/time.

Reply:
done